packages/mom5/package.py

```diff
@@ -59,14 +67,18 @@ def edit(self, spec, prefix):
         makeinc_path = join_path(srcdir, "bin", "mkmf.template.spack")
         config = {}
 
-        istr = join_path((spec["oasis3-mct"].headers).cpp_flags, "psmile.MPI1")
         # NOTE: The order of the libraries matters during the linking step!
         if "@access-esm1.5" in self.spec:
-            ideps = ["oasis3-mct"]
+            istr = " ".join([
+                    join_path((spec["oasis3-mct"].headers).cpp_flags, "psmile.MPI1"),
+                    join_path((spec["oasis3-mct"].headers).cpp_flags, "pio"),
+                    join_path((spec["oasis3-mct"].headers).cpp_flags, "mct")])
+            ideps = ["netcdf-fortran"]
             ldeps = ["oasis3-mct", "netcdf-c", "netcdf-fortran"]
-            FFLAGS_OPT = "-O3 -debug minimal -xCORE-AVX2"
+            FFLAGS_OPT = "-O3 -debug minimal -march=cascadelake -xCORE-AVX512 -align array64byte"
```

Comment:
Is the `-march=cascadelake` actually required?

Reply:
Removed in all instances. Please check.